src/new-client/src/App/LiveRates/Tile/Tile.tsx

```diff
@@ -132,4 +157,171 @@ const TileContext: React.FC<{
   },
 )
 
+const history = generateHistoricPrices(30)
+const dataPoints = getDataPoints<HistoryPrice>((price, idx) => [
+  new Date(idx),
+  price.mid,
+])(history)
+const scales = withScales([0, 125], [0, 75])(dataPoints)
+const HistoryMockSvgPath = toSvgPath(curveBasis)(scales)
+
+export const TileMockComponent: React.FC<MockProps> = ({
```

Comment:
This was always going to be the difficult one.
I think creating your own mock component, composing the existing components that make up the Tile, is a good idea.
However, I think it would be good to keep this isolated from the actual Tile component.
Can we move it (and anything else supporting it) to it's own file?

Reply:
[Fixed here](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1970/commits/bebfc79ca887b2478f776dfaba36f2ce1f214d5c), I added this data in the `SpotTilesMockData` file so it doesn't polute the `Tile.tsx`